optaweb-employee-rostering-backend/src/main/resources/solverConfig.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <solver>
   <scoreDirectorFactory>
-    <scoreDrl>org/optaweb/employeerostering/service/solver/employeeRosteringScoreRules.drl</scoreDrl>
-    <!--<constraintProviderClass>org.optaweb.employeerostering.service.solver.EmployeeRosteringConstraintProvider</constraintProviderClass>-->
+    <!--<scoreDrl>org/optaweb/employeerostering/service/solver/employeeRosteringScoreRules.drl</scoreDrl>-->
+    <constraintProviderClass>org.optaweb.employeerostering.service.solver.EmployeeRosteringConstraintProvider</constraintProviderClass>
```

Comment:
Got any numbers for me? How much better/worse are we now?

Reply:
Drools - Constraint Stream performance is almost 1-to-1 according to the benchmark:
```
Drools: Average: 26,411/s Problem 0: 16,665/s Problem 1: 36,158/s
CS     : Average: 25,830/s Problem 0: 16,596/s Problem 1: 35,065/s
```

Reply:
(This is comparing Quarkus Drools to Quarkus Constraint Streams; need to rerun on Spring Boot)

Reply:
Default on Spring boot is faster than both:
```
Spring-Boot Drools: 
Average: 29,662/s Problem 0: 19,758/s  Problem 1: 39,567/s
```

Reply:
CS on Spring boot is also faster than both:
```
Spring-Boot Constraint Stream
Average: 29,105/s Problem 0: 21,401/s  Problem 1: 36,809/s
```

Reply:
Benchmark: Default Solver, warm up 30 seconds, 10 seconds spent limit

Reply:
OK, that is good news.